### PR TITLE
Removes Cringe Ion Storm Laws

### PR DESCRIPTION
**Pull request by DorainGrey**
Removes two OCK terms from the potential Ion Laws.
There's also an ion law that states the AI must "be read Woody's Got Wood to". Not sure about whether to keep that one, so I've left it out. It does essentially valid the AI but it's also kind of amusing.

### CODE REVIEW

File: vgstation13.dme [hunk 1]
```diff
@@ -2828,7 +2828,8 @@
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
 #include "maps\_map_override.dm"
-#include "maps\tgstation.dm"
+#include "maps\roidstation.dm"
```
Comment (Jack7D1) [inline vgstation13.dme]:
```suggestion
#include "maps\tgstation.dm"
```

File: vgstation13.dme [hunk 2]
```diff
@@ -2828,7 +2828,8 @@
...
+#include "maps\RoidStation.dmm"
```
Comment (Jack7D1) [inline vgstation13.dme]:
```suggestion
```

**Review by Jack7D1 — CHANGES_REQUESTED** [review]
Be careful!

**Comment by DorainGrey** [comment]
> Be careful!

Thanks, I tend to use git add, for the specific file I was doing but this snuck in as I did this in 20 seconds.

**Comment by gurfan** [comment]
angry valids made this pr

**Comment by DorainGrey** [comment]
> angry valids made this pr

I really don't care I died, AI's can still easily get ion laws to kill a specific person.
This just removes the confusion.

**Comment by jwhitak** [comment]
Giving the AI confusing terms in their ion laws is fun! I can see where the logic of this change is coming from, but it removes the soul...

**Comment by DorainGrey** [comment]
> Giving the AI confusing terms in their ion laws is fun! I can see where the logic of this change is coming from, but it removes the soul...

I don't think OOC terms in AI laws are soul but that's just me. It's just banbait for AI's
Plenty of confusing possibilities left.

**Comment by Eneocho** [comment]
Sovlless

**Comment by Exxion** [comment]
I have actually gotten banned on this very server for using this word

**Comment by DorainGrey** [comment]
Right I give up, inb4 this starts another argument over silicon laws in 6 months or so.

**Comment by DorainGrey** [comment]
I give up, it'd be nice if the admemes at least gave us a clear ruling on what valid means in an IC context before closing it though.

**Comment by DamianX** [comment]
"valid" being a _valid_ adjective to describe a person (the first thing that was r\*moved in this PR) is okay: the word itself can be used to mean a lot of different things. "Time to get those valids" has an established OOC meaning, and it would be best to keep it out of in-character baked-in laws.
Also, r\*move silicons.

**Comment by WindowsErrors** [comment]
cringe?

**Comment by DorainGrey** [comment]
Closing due to lack of interest. Just watch this be a problem again in 6 months.

**Pull request closed without merging**